### PR TITLE
Namsonx/task/stabi branch

### PR DESCRIPTION
**Pull request by namsonx**

### CODE REVIEW

File: packagedoc/additional_docs/History.tex [hunk 1]
```diff
@@ -46,4 +46,10 @@
 \historyversiondate{0.7.5}{03/2024}
 \historychange{- Simplify local configuration loading based on the new version of JsonPreprocessor}
 
+\historyversiondate{0.7.6}{04/2024}
+\historychange{Validates parameter names which are loaded from configuration files:\newline
+- Verify all elements in params['global'] which are set as global variables of Robot run.\newline
+- Parameter names must start with a letter or underscore, followed by letters, digits, or underscores.\newline
+- Ensure that the variable name does not conflict with reserved keywords.}
+
```
Comment (HolQue) [inline packagedoc/additional_docs/History.tex:52]: Hi Son,

this is so much long winded wording and contains too much technical details for an entry in history.

That's really enough:

`Added naming convention check for parameter names.`

To explain the rules behind the naming convention is for the documentation.

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi  @namsonx ,
can you please update the documentation accordingly.
Release_info update is missing, too.

btw. please provide as soon as possible also a release info for the jsonpreprocessor stabi branch. 
Merge: https://github.com/test-fullautomation/python-jsonpreprocessor/pull/254 had no release info.

release_info must have enduser perspective.
e.g. improved and more detailed error messages, short examples of jpp code which is supported now, ...

Thank you,
 Thomas

**Review by HolQue — CHANGES_REQUESTED** [review]
Hi Son,

in https://github.com/test-fullautomation/robotframework-testsuitesmanagement/issues/238
I mentioned only one single example. Please go through your code and fix all positions with manual line breaks.

**Comment by namsonx** [comment]
> Hi Son,
> 
> in #238 I mentioned only one single example. Please go through your code and fix all positions with manual line breaks.

Hello Holger,

I updated all newline cases in error message log.

Thank you,
Son

**Comment by HolQue** [comment]
Hi Son,

still line breaks! But they are caused by the JsonPreprocessor now. Probably in lines like;

`raise Exception(f"JSON file: {jFile}\n{error}")`

Therefore the JsonPreprocessor also needs to be fixed.


For example:

```
20240318 14:16:18.915 - ERROR - Loading of JSON configuration file failed! 
          Reason: JSON file: D:/ROBFW/Dev/GenSnippetsTSM/tmp_files/config/testconfig.json
Expecting property name enclosed in double quotes: line 15 column 28 (char 398)
20240318 14:16:18.915 - UNKNOWN - Loading of JSON configuration file failed!
```

This really does not look nice.

**Comment by test-fullautomation** [comment]
Hi @namsonx ,
can you please fix the complete line-break issue. 
Also:
can you please update the documentation accordingly.
Release_info update is missing, too.

Please also check why the build was failing.

Then I will merge.

Thank you,
 Thomas

File: packagedoc/additional_docs/History.tex (hunk 1 above)
Comment (namsonx) [reply packagedoc/additional_docs/History.tex:52]: Hello Holger,

Thank you for your suggestion. I updated

**Comment by HolQue** [comment]
Hi Son,

please consider also

https://github.com/test-fullautomation/robotframework-testsuitesmanagement/issues/250

for update of history in component documentation.

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
looks good to me now!
Thank you,
 Thomas

**Pull request merged**